src/workerd/io/actor-cache.c++

```diff
@@ -577,6 +582,7 @@ kj::OneOf<ActorCache::GetResultList, kj::Promise<ActorCache::GetResultList>> Act
   {
     auto lock = lru.cleanList.lockExclusive();
     for (auto& key: keys) {
+      ActorStorageLimits::checkMaxKeySize(key); // new, can't find where this check is done for getMultiple, should I leave this out?
```

Comment:
Will delete this comment, but yeah, I think this is new, so should I leave this check out since it wasn't being done previously as far as I can tell?

Reply:
You should leave this check out because it's not backwards compatible.

That being said, we probably want to add this check.  In a follow-up PR, we can add it as a compatibility date update.

It's weird that we don't call checkMaxPairsCount() for putMultiple and deleteMultiple.  According to our [docs](https://developers.cloudflare.com/durable-objects/api/transactional-storage-api/), get, put, and delete are all supposed to have a 128 entry limit.

Reply:
:+1: removed this check for now.